recipe/meta.yaml

```diff
@@ -51,19 +48,21 @@ requirements:
     - rasterio =1.3
     - requests =2.31
     - salib =1.4
-    - scikit-learn =1.3
-    - scipy =1.11
-    - sparse =0.14
+    - scikit-learn =1.4
+    - scipy =1.12
+    - sparse =0.15
     - statsmodels =0.14
     - tabulate =0.9
     - tqdm =4.66
     - unittest-xml-reporting =3.2
-    - xarray =2023.8
+    - xarray =2024.1
     - xlrd =2.0
     - xlsxwriter =3.1
-    - xmlrunner =1.7
-    # TODO: Remove with next release
-    - xyzservices =2023.7  # Fix old version to avoid error on import
+    - deprecation =2.1.0
+    - overpy =0.7
+    - peewee =3.17.1
+    - pillow =9.4.0
+    - xmlrunner =1.7.7
```

Comment:
🧐 no, of course we don't. I wonder how they came in.
(I'm running a script for updating meta.yaml. Obviously it needs some corrections. Also for the trailing white spaces!)

Reply:
Do we need to specify these packages at all? They were not necessary before 🤔

Reply:
These are the packages that are listed in `install_requires` from setup but not in requirements/env_climada.yml. Hence I guess they are provided by PyPi and not by Conda-Forge.
Since we don't specify their versions in setup, I think it makes sense to specify their version here. Why not?

Reply:
> Correction: peewee and pillow were moved. We should put them in alphabetical order again. overpy, deprecation, and numexpr were not listed before.
Ok. That's enough. I really need to fix that script!

Reply:
> These are the packages that are listed in install_requires from setup but not in requirements/env_climada.yml. 

Alright, then we should indeed list them here. I also got confused by the change in order, apparently the only new package in the requirements list is `numexpr`. So, all good! Thanks for double-checking!